yt_chat/utils/qdrant.py

```diff
@@ -0,0 +1,15 @@
+from qdrant_client.http.models import Distance, VectorParams
+
+from qdrant_client import QdrantClient
+
+def create_qdrant_collection(collection_name: str, embedding_vector_size: int) -> QdrantClient:
+    qdrant_client = QdrantClient(":memory:")
+    # qdrant_client = QdrantClient(path="path/to/db")  # Persists changes to disk
```

Comment:
Comments to be deleted

Reply:
I'd like to keep them so that we can make a new PR giving the possibility to choose between memory, disk, and URL